test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,79 @@
+package syncer
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	apb "github.com/kcp-dev/kcp-tests/test/extended/apibinding"
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate create, modifying, deleting a deployment from KCP gets synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("None pcluster kubeconfig set for the test scenario")
+		}
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
+
+		g.By("# Create workload sync and generate syncer resources manifests in current workspace")
+		mySyncer := NewSyncTarget()
+		mySyncer.OutputFilePath = "/tmp/" + myWs.Name + "." + mySyncer.Name + ".yaml"
+		mySyncer.Create(k)
+		defer mySyncer.Clean(k)
+
+		g.By("# Apply syncer resources on pcluster and wait for synctarget become ready")
+		defer k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("-f", mySyncer.OutputFilePath).Execute()
+		err := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("apply").Args("-f", mySyncer.OutputFilePath).Execute()
+		o.Expect(err).ShouldNot(o.HaveOccurred())
+		mySyncer.WaitUntilReady(k)
+		mySyncer.CheckDisplayAttributes(k)
+
+		g.By("# Create a new workspace and create an APIBinding to attach the compute to the new workspace")
```

Comment:
@Phaow I do not think we need to create new workspace and apibinding. There exists an apibinding when synctarget is added to the ws. Once the synctarget is ready we could simply create workload and check if that runs on kcp also runs fine on pcluster.

Also the g.By should be something like "create a workload and verify that it runs on kcp & gets synced to pcluster)

Reply:
You're right. Hmm. I think this is a checkponit for this test scenario(just follow the polarion steps). Previously I just use the same ws works well. I could remove the step  and also update the polarion if needed.

Reply:
@Phaow not sure what polarion steps you are referring to but the case OCP-50877 in polarion does not have steps to create a new apibinding, just checked now.

Reply:
@kasturinarra Sorry, my bad.  The step is from our testplan not sure if we could remove the step directly.
https://docs.google.com/document/d/1BosHAdSi34S-UXv1ksr2e0lkmNDJ2ZPpIcBkpqqGoAg/edit

Reply:
@Phaow not sure if kind cluster requires that, but when we add OCP cluster as sync target we do not need this. You could remove it  and i do not see any issues with that.

Reply:
Yeah, for kind clusters the step is also optional. I'll remove it. We've already have an ApiBinding case covered the step checkpoint.

Reply:
Addressed.